mapbox/libjava-services/src/test/java/com/mapbox/services/api/directions/v5/MapboxDirectionsTest.java

```diff
@@ -168,27 +171,30 @@ public void noProfileProvided() throws ServicesException {
   }
 
   @Test
-  public void requiredCoordinates() throws ServicesException {
+  public void checksCoordinatesAreRequired() throws ServicesException {
     thrown.expect(ServicesException.class);
     thrown.expectMessage(startsWith("You should provide at least two coordinates (from/to)"));
+
     new MapboxDirections.Builder().setAccessToken("pk.XXX").setProfile(DirectionsCriteria.PROFILE_DRIVING).build();
   }
 
   @Test
-  public void testSanity() throws ServicesException, IOException {
+  public void checksSanity() throws ServicesException, IOException {
     MapboxDirections client = new MapboxDirections.Builder()
       .setAccessToken("pk.XXX")
       .setCoordinates(positions)
       .setProfile(DirectionsCriteria.PROFILE_DRIVING)
       .setBaseUrl(mockUrl.toString())
       .build();
+
     Response<DirectionsResponse> response = client.executeCall();
-    assertEquals(response.code(), 200);
-    assertEquals(response.body().getCode(), DirectionsCriteria.RESPONSE_OK);
+
+    assertEquals(200, response.code());
+    assertEquals("Ok", response.body().getCode());
```

Comment:
@zugaldia Ok agreed. It's also explanatory using the constant so I leave it as it was 👍